airbyte-integrations/connectors/source-hubspot/source_hubspot/helpers.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import abc
+from typing import MutableMapping
+
+
+class IRecordPostProcessor(abc.ABC):
+    @abc.abstractmethod
+    def add_record(self, record: MutableMapping, primary_key: str = None):
```

Comment:
can we add a one-liner so we remember what the interface is for

Reply:
sure, done